.vsts-ci.yml

```diff
@@ -10,21 +10,15 @@ variables:
   BuildConfiguration: Release
   BuildPlatform: Any CPU
 
-resources:
-  containers:
-  - container: xenial
-    image: andrewarnott/linux-buildagent
-
 jobs:
 - job: Windows
-  pool: Hosted VS2017
+  pool: Hosted Windows 2019 with VS2019
   steps:
   - template: azure-pipelines/build.yml
 
 - job: Linux
   pool:
     vmImage: Ubuntu 16.04
-  container: xenial
```

Comment:
Because the `UseDotNet@2` task in Azure Pipelines finally works properly for Linux. Also because .NET Core 1.0 couldn't be installed on certain flavors/versions of Linux so we had to be very controlling as to which one we got so we knew it would work. Now these are both non-issues.
By removing the container requirement, we shave a couple minutes off of the linux build agent time since it doesn't have to download an image before starting.